core/systems/RessourceManager.gd

```diff
@@ -0,0 +1,31 @@
+extends Node2D
+
+class_name RessourceManager
+
+@export var humans = 0
```

Comment:
anzahl der humans werden auch in der HumanFactory gemanaged, da sollten wir dann noch anpassen, dass wir das nicht zwei mal tracken
oder sind mit diesen humans die medics etc. gemeint?
humans aus HumanFactory sind die "kranken" NPCs

Reply:
Das sind die Ressourcen, also die Humans die ich schon fertig angeklickt habe, um diese später auszugeben für Gebäude

Reply:
alright!